docs/src/InvariantTheory/it.md

```diff
@@ -0,0 +1,52 @@
+```@meta
+CurrentModule = Oscar
+```
+
+```@setup oscar
+using Oscar
+```
+
+```@contents
+Pages = ["it.md"]
+```
+
+# Introduction
+
+From a theoretical point of view, our basic setting in invariant theory consists of a group $G$ which acts on a vector space $V$ of finite
+dimension $n$ over a field $K$. The action of $G$ on $V$ induces actions of $G$ on the dual vector space $V^\ast$,
+
+$(\pi \cdot f)(v)=f(\pi^{-1}\cdot v),$
+
+and, thus,  on the graded symmetric algebra 
+
+$K[V]:=S(V^*)=\bigoplus_{d\geq 0} S^d V^*.$
+
+The fixed points of this action are the invariants of $G$, and the graded subalgebra
+
+$K[V]^G=\{f\in K[V] \mid \pi\cdot  f=f {\text { for any }} \pi\in G\}\subset K[V]$
+
+is its ring of invariants.
+
+!!! note
+    Except where mentioned otherwise, we will be in the favourable situation where $G$ is a linear reductive group which acts rationally on $V$. This has several important consequences:
```

Comment:
```suggestion
    Except where mentioned otherwise, we will assume to be in the favourable situation where $G$ is a linear reductive group which acts rationally on $V$. This has several important consequences:
```

Reply:
I disagree! In a textbook, we assume, in practical applications we are or we are not in the favourable situation ...